java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -4431,4 +4435,66 @@ public void getbit_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(bit, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void lmpop_returns_success() {
+        // setup
+        String key = "testKey";
+        String[] keys = {key};
```

Comment:
please use more than a single key in your test to verify that we push all (not just the first) key.